src/components/Images.vue

```diff
@@ -0,0 +1,283 @@
+      <input
+        v-model="imageToBuild"
+        :disabled="showImageManagerOutput"
+        type="text"
+        maxlength="50"
+        placeholder="image name with tag"
+        class="input-sm inline"
+      >
+      <button
+        class="btn btn-sm role-tertiary"
+        :disabled="imageToBuildButtonDisabled"
+        @click="doBuildAnImage"
+      >
+        Build an Image...
+      </button>
+      <hr>
+      <div v-if="showImageManagerOutput">
+        <textarea
+          id="imageManagerOutput"
+          ref="outputWindow"
+          v-model="imageManagerOutput"
+          rows="10"
+        />
+        <button
+          v-if="imageManagerProcessIsFinished"
+          @click="closeTheOutputWindow"
+        >
+          Close This Output
+        </button>
+      </div>
+    </div>
+    <div v-else>
+      <p>Waiting for Kubernetes to be ready</p>
+    </div>
+  </div>
+</template>
+
+<script>
+import ButtonDropdown from '@/components/ButtonDropdown';
+import SortableTable from '@/components/SortableTable';
+import Checkbox from '@/components/form/Checkbox';
+
+const { ipcRenderer } = require('electron');
+const K8s = require('../k8s-engine/k8s');
+
+export default {
+  components: {
+    ButtonDropdown, Checkbox, SortableTable
+  },
+  props:      {
+    images: {
+      type:     Array,
+      required: true,
+    },
+    k8sState: {
+      type:    Number,
+      default: K8s.State.STOPPED,
+    },
+    showAll: {
+      type:    Boolean,
+      default: false,
+    },
+  },
+
+  data() {
+    return {
+      kimRunningCommand: null,
+      headers:           [
+        {
+          name:  'imageName',
+          label: 'IMAGE',
+          sort:  ['imageName', 'tag', 'imageID'],
+        },
+        {
+          name:  'tag',
+          label: 'TAG',
+          sort:  ['tag', 'imageName', 'imageID'],
+        },
+        {
+          name:  'imageID',
+          label: 'IMAGE ID',
+          sort:  ['imageID', 'imageName', 'tag'],
+        },
+        {
+          name:  'size',
+          label: 'SIZE',
+          sort:  ['size', 'imageName', 'tag'],
+        },
+      ],
+      imageToBuild:                     '',
+      imageToPull:                      '',
+      imageManagerOutput:               '',
+      keepImageManagerOutputWindowOpen: false,
+      fieldToClear:                     null,
+    };
+  },
+  computed: {
+    rows() {
+      if (this.showAll) {
+        return this.images;
+      }
+
+      return this.images.filter(this.isDeletable);
+    },
+    k8sIsRunning() {
+      return this.k8sState === K8s.State.STARTED;
+    },
+    showImageManagerOutput() {
+      return !!this.kimRunningCommand || this.keepImageManagerOutputWindowOpen;
+    },
+    imageManagerProcessIsFinished() {
+      return !this.kimRunningCommand;
+    },
+    imageToBuildButtonDisabled() {
+      return this.showImageManagerOutput || this.imageToBuild.length === 0 || !this.imageToBuild.includes(':');
```

Comment:
```suggestion
      return this.showImageManagerOutput || !this.imageToBuild.includes(':');
```
If a string includes `:` then it can't be empty.

Reply:
True dat